.travis.yml

```diff
@@ -3,65 +3,29 @@ sudo: true
 language:
   - cpp
 
-env:
-  global:
-    - secure: "ReYURSvDLRz078njJgh2i/KvwLc/3fGPkgjf6mLLoHWgz7vSuh24B5z10Th2UMQo9op66qcZ2WcooYIqeEHGbmYuBGGKd2axsGUEXqoclG9SdNsa5BoCA9iiLlLiMgIO+Kp1EuSbWcrpVRs/BiXya1iN+lpBErsWHGaXJP8idexnCepMLYgzUDu/FNuZJwRHMX1cvqK7wxOkCwXJRqEfbA+fn1/oCb+YAPpEet12kFNu1RRk6g+ui7g5XuAVHISj+UHbr5+GNYR6gD8476EkjUm38ssnPKxTvydj1QXcEw6weRDpevFByum5fDf2vHgojo5dNvhIio4gENkfbwbd2azVitHtE7boByb/AO3z3YcFhDRu0ETgCEZWIDrQWwkAubf+sB2MWyKGCPGQaECrVGDD2JzV6kX0lMCVaPQONggFBmJQwcHodLxXurpupmttJ+W4aZf5EsQsGPIYNegMnkdHAgTHxuPUtyDIlOJfTrUt1qPRh+yKGvmX7/C+AirXFIbxOemTdx3w0Y3CbhVxP2jDoP45rPzC17EI06gdQJaWzx5//LKkwcsAmox0bXrKHDOuMJNWVA6ZhBd3fGascMh4tgdSYc9yuRyAb0ygP0obUXp6PXU0TFxeenCaA9YAZrg6OXzS115lcnak8Xfyi+fmlfjrT8BSVk9bL9/3t2I="
-
 matrix:
   include:
-    - os: linux
-      env:
-        - BACKEND=OpenBLAS
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - libopenblas-base
-            - libopenblas-dev
-    - os: linux
-      env:
-        - BACKEND=ATLAS
-      addons:
-        sources:
-          - ubuntu-toolchain-r-test
-        apt:
-          packages:
-            - libatlas-base-dev
+    #########
+    # Linux Machines
+    #########
     - os: linux
       env:
         - BACKEND=Intel10_64lp_seq
-        - BUILD_STATIC=1
+        - BUILD_STATIC=OFF
     - os: linux
       env:
         - BACKEND=Intel10_64lp_seq
+        - BUILD_STATIC=ON
     - os: linux
       env:
         - BACKEND=Intel10_64lp
-        - BUILD_STATIC=1
+        - BUILD_STATIC=OFF
     - os: linux
       env:
         - BACKEND=Intel10_64lp
-    - os: osx
-      env:
-        - BACKEND=Apple
-    - os: osx
-      env:
-        - BACKEND=Intel10_64lp_seq
-    - os: osx
-      env:
-        - BACKEND=Intel10_64lp_seq
-        - BUILD_STATIC=1
-    - os: osx
-      env:
-        - BACKEND=Intel10_64lp
-    - os: osx
-      env:
-        - BACKEND=Intel10_64lp
-        - BUILD_STATIC=1
-    - os: osx
-      env:
-        - BACKEND=OpenBLAS
```

Comment:
please put back the mac mkl builds

Reply:
piing.

please put back the mac builds here so we can move on.

Reply:
but not the OpenBLAS one.

Reply:
my idea was that once everything was green remove the `aed69c6` commit.